news/_posts/releases/2021-08-31-prestashop-1-7-7-8-maintenance-release.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.7.8"
+subtitle: "Maintenance version of the 1.7.7.x branch"
+date:   2021-08-19
+authors: [ PrestaShop ]
+image: /assets/images/2021/08/1.7.7.8_build.png
+icon: icon-leaf
+tags:
+ - version
+ - maintenance
+ - patch
+ - releases
+ - 1.7.7.x
+ - 1.7
+
+---
+
+PrestaShop 1.7.7.8 is now available. This maintenance release fixes 1 major regression.
+
+![1.7.7.8 is available!](/assets/images/2021/08/1.7.7.8_build.png)
+
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before.
+
+## Major issues on the product page
+
+- The issue [#25718](https://github.com/PrestaShop/PrestaShop/issues/25718) and the issue [#25753](https://github.com/PrestaShop/PrestaShop/issues/25753) appeared due to the implementation of the new numeric data format for number fields on the 1.7.7.6 patch version. This regression causes users to be unable to edit the impact of a combination and rounding problems on the impact calculation. 
```

Comment:
```suggestion
- The issue [#25718](https://github.com/PrestaShop/PrestaShop/issues/25718) and [#25753](https://github.com/PrestaShop/PrestaShop/issues/25753) appeared due to the implementation of the numeric data format for number fields on the 1.7.7.6 patch version. This regression prevents users to edit the impact of a combination and causes rounding problems on the impact calculation. 
```
I suggest to remove "issue" to avoid a repetition and "new" that is already suggested by "implementation"